crates/release-automation/README.md

```diff
@@ -69,70 +110,243 @@ Automated steps still require running the tool manually ;-).
     0. Create a commit with the version and changelog changes
     0. Create a tag for each crate release (***NOTE***: This is likely subject to change because it creates issues in case of publish failures later on. It would probably be preferable to only create tags after a successful publish.)
 
-    The command for this is:
+    The commands for this are:
 
     ```sh
-    hc-ra \
+    nix-shell --run '
+      hc-ra \
+        --workspace-path=$PWD \
         --log-level=info \
         release \
-            --dry-run \
-            --match-filter="^(holochain|holochain_cli|kitsune_p2p_proxy)$" \
-            --disallowed-version-reqs=">=0.1" \
-            --allowed-matched-blockers=UnreleasableViaChangelogFrontmatter \
-            --steps=CreateReleaseBranch,BumpReleaseVersions'
+          --steps=CreateReleaseBranch
+
+      hc-ra \
+        --workspace-path=$PWD \
+        crate apply-dev-versions --commit
+
+      hc-ra \
+        --workspace-path=$PWD \
+        --log-level=info \
+        release \
+          --dry-run \
+          --match-filter="^(holochain|holochain_cli|kitsune_p2p_proxy)$" \
+          --disallowed-version-reqs=">=0.1" \
+          --steps=BumpReleaseVersions
+      '
     ```
 
     If this succeeds, repeat the command without the `--dry-run` to perform the changes.
 
-0. _[M]_ Push the release branch. Example:
+    ***NOTE***: If at any point after this any changes need to be made to the code for this release, please come back here and follow these steps:
+
+    0. Drop the commit made by the _BumpReleaseVersions_ step using `git rebase -i HEAD~1`.
+    0. Make the required changes and commit them.
+    0. Repeat the _BumpReleaseVersions_ step.
+    0. Continue the process from there...
+
+0. _[M]_ Add additional release-specific environment variables:
 
     ```sh
-    git push -u upstream $(git branch --show-current)
+    export TAG=$(git -C ${HOLOCHAIN_REPO} tag --list | grep holochain- | tail -n1)
+    export VERSION=${TAG/holochain-/}
+    export VERSION_COMPAT="v${VERSION//./_}"
+    export RELEASE_BRANCH=$(git branch --show-current)
     ```
 
-0. _[M]_ Create a Pull-Request from the release branch to the main branch
+0. _[M]_ Push the release branch. Example:
+
+    ```sh
+    git push -u origin $(git branch --show-current)
+    ```
 
-0. _[M]_ Ensure the CI tests pass
+0. _[M]_ Create a Pull-Request from the release branch to the main branch and ensure the CI tests pass
```

Comment:
Although it doesn't say `main` branch but main branch, it's clearer to refer to the `develop` branch.

Reply:
Nevermind.